pkg/webui/console/containers/header/index.js

```diff
@@ -152,6 +152,12 @@ class Header extends Component {
             path="/user/api-keys"
           />
         )}
+        <Dropdown.Item
+          title={sharedMessages.profileSettings}
+          icon="user"
+          path="/oauth/profile-settings"
```

Comment:
I think we should get the account app prefix from is configuration. Similar to how its done in the account app when navigating to the Console, see https://github.com/TheThingsNetwork/lorawan-stack/blob/v3.12/pkg/webui/account/views/overview/index.js#L45.

cc @kschiffer

Reply:
Yeah indeed. Please add a config option to the console `--console.ui.account-url` (defaulting to `/account`) and source the prefix from there.

See the following files to add this:

[https://github.com/TheThingsNetwork/lorawan-stack/blob/v3.12/pkg/console/config.go#L41-L46](https://github.com/TheThingsNetwork/lorawan-stack/blob/v3.12/pkg/console/config.go#L41-L46)

[https://github.com/TheThingsNetwork/lorawan-stack/blob/v3.12/cmd/internal/shared/console/config.go#L45-L47](https://github.com/TheThingsNetwork/lorawan-stack/blob/v3.12/pkg/console/config.go#L41-L46https://github.com/TheThingsNetwork/lorawan-stack/blob/v3.12/cmd/internal/shared/console/config.go#L45-L47)

And then please add a Console scoped env selector in here: [https://github.com/TheThingsNetwork/lorawan-stack/tree/v3.12/pkg/webui/console/lib](https://github.com/TheThingsNetwork/lorawan-stack/tree/v3.12/pkg/webui/console/lib)

Similar to the one of the Account App: https://github.com/TheThingsNetwork/lorawan-stack/tree/v3.12/pkg/webui/account/lib/selectors